diff --git a/Source/CMakeLists.txt b/Source/CMakeLists.txt
--- a/Source/CMakeLists.txt
+++ b/Source/CMakeLists.txt
@@ -1052,7 +1052,6 @@ if(CMake_HAVE_CXX_ATOMIC_LIB)
 # On Apple we need CoreFoundation and CoreServices
 if(APPLE)
   target_link_libraries(CMakeLib PUBLIC "-framework CoreFoundation")
-  target_link_libraries(CMakeLib PUBLIC "-framework CoreServices")
 endif()
 
 if(WIN32 AND NOT UNIX)
diff --git a/Source/cmGlobalXCodeGenerator.cxx b/Source/cmGlobalXCodeGenerator.cxx
--- a/Source/cmGlobalXCodeGenerator.cxx
+++ b/Source/cmGlobalXCodeGenerator.cxx
@@ -61,10 +61,6 @@
 #  include <CoreFoundation/CFString.h>
 #  include <CoreFoundation/CFURL.h>
 #  include <CoreFoundation/CFUUID.h>
-#  if !TARGET_OS_IPHONE
-#    define HAVE_APPLICATION_SERVICES
-#    include <ApplicationServices/ApplicationServices.h>
-#  endif
 #endif
 
 #if !defined(CMAKE_BOOTSTRAP)
diff --git a/Utilities/cmlibarchive/CMakeLists.txt b/Utilities/cmlibarchive/CMakeLists.txt
--- a/Utilities/cmlibarchive/CMakeLists.txt
+++ b/Utilities/cmlibarchive/CMakeLists.txt
@@ -2180,10 +2180,6 @@ IF(ENABLE_TEST)
   ADD_CUSTOM_TARGET(run_all_tests)
 ENDIF(ENABLE_TEST)
 
-# We need CoreServices on Mac OS.
-IF(APPLE)
-  LIST(APPEND ADDITIONAL_LIBS "-framework CoreServices")
-ENDIF(APPLE)
 
 add_subdirectory(libarchive)
 IF(0) # CMake does not build libarchive's command-line tools.
