diff --git a/Source/cmFileCopier.cxx b/Source/cmFileCopier.cxx
--- a/Source/cmFileCopier.cxx
+++ b/Source/cmFileCopier.cxx
@@ -41,7 +41,7 @@ cmFileCopier::MatchProperties cmFileCopier::CollectMatchProperties(
   const std::string& file)
 {
   // Match rules are case-insensitive on some platforms.
-#if defined(_WIN32) || defined(__APPLE__) || defined(__CYGWIN__)
+#if defined(_WIN32) || defined(__APPLE__)
   const std::string file_to_match = cmSystemTools::LowerCase(file);
 #else
   const std::string& file_to_match = file;
diff --git a/Source/kwsys/SystemTools.cxx b/Source/kwsys/SystemTools.cxx
--- a/Source/kwsys/SystemTools.cxx
+++ b/Source/kwsys/SystemTools.cxx
@@ -560,9 +560,6 @@ public:
     PathCaseMap;
   std::map<std::string, std::string> EnvMap;
 #endif
-#ifdef __CYGWIN__
-  StringMap Cyg2Win32Map;
-#endif
 
   /**
    * Actual implementation of ReplaceString.
